### ISSUE TITLE
Terminal doesn't accept any keyboard input

### ISSUE THREAD

**Issue by Kerosz** [microsoft/terminal#5643]
After reinstalling the windows yesterday, switched my primary drive to another one, the terminal doesn't take any keyboard input, I cannot write anything in it. I can use the mouse to select and paste things into it. 

Something I have tried is to uninstall and reinstall the terminal but it has the same behaviour, JSON settings are default.
Tried PowerShell and CMD separately and they work fine.

Any idea of what could cause this?

**Comment by zadjii-msft** [comment]
@Kerosz What version of the Terminal are you running?

Are you using PowerToys?

By any chance is what you're seeing similar to #4448?

**Comment by Kerosz** [comment]
Windows Terminal  0.11.1191.0
OS Build: 19041.208 on Windows PRO for Workstations

No PowerToys used as it was a clean reinstall of windows and I didn't run the setup scripts yet.

Managed to fix the issue by going back to a previous version of windows using the recovery and updating again to the latest version. It was a 20 minutes process and it actually fixed the issue.

No idea what could have caused it as in both cases I just installed the latest version of W and got the WT from the store.
